src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,160 @@
+- More efficient pipeline for security fixes during the CVE pre-disclosure process.
+
+## Dev instructions
+
+Please note that **this repository is an experiment and a work-in-progress so it is possible that the build is broken**.
+For the latest information about Source-Build support, please watch for announcements posted on our [GitHub Discussions page](https://github.com/dotnet/source-build/discussions).
+
+### Prerequisites
+
+The dependencies for building .NET from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md).
+
+### Building
+
+1. **Clone the VMR**
+
+   ```bash
+   git clone https://github.com/dotnet/dotnet dotnet-dotnet
+   ```
+
+2. **Prep the source to build on your distro**  
+   This downloads a .NET SDK and a number of .NET packages needed to build .NET from source.
+
+    ```bash
+    cd dotnet-dotnet
+    ./prep.sh
+    ```
+
+3. **Build the .NET SDK**
+
+    ```bash
+    ./build.sh --clean-while-building --online
+    ```
+
+    This builds the entire .NET SDK from source.
+    The resulting SDK is placed at `artifacts/x64/Release/dotnet-sdk-8.0.100-your-RID.tar.gz`.
+
+    Currently, the `--online` flag is required to allow NuGet restore from online sources during the build.
+    This is useful for testing unsupported releases that don't yet build without downloading pre-built binaries from the internet.
+
+    Run `./build.sh --help` to see more information about supported build options.
+
+4. *(Optional)* **Unpack and install the .NET SDK**
+
+    ```bash
+    mkdir -p $HOME/dotnet
+    tar zxf artifacts/x64/Release/dotnet-sdk-8.0.100-your-RID.tar.gz -C $HOME/dotnet
+    ln -s $HOME/dotnet/dotnet /usr/bin/dotnet
+    ```
+    
+    To test your source-built SDK, run the following:
+
+    ```bash
+    dotnet --info
+    ```
+
+## List of components
+
+To enable full offline source-building of the VMR, we have no other choice than to synchronize all the necessary code into the VMR. This also includes any code referenced via git submodules. More details on why we need to do and how we do that can be found here:
+- [Strategy for managing external source dependencies](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Strategy-For-External-Source.md)
+- [Source Synchronization Process](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#source-synchronization-process)
+
+### Detailed list
+
+<!-- component list beginning -->
+
+> Auto-generated list of components will go here
+
+<!-- component list end -->
+
+The repository also contains a [JSON manifest](https://github.com/dotnet/dotnet/blob/main/src/source-manifest.json) listing all components in a machine-readable format.
+
+## Filing Issues
+
+This repo should only contain issues that are tied to the VMR construction/build itself. For other issues, please file them to their appropriate development repos.
```

Comment:
> This repo should only contain issues that are tied to the VMR construction/build itself.

I thought we were going to hold off on enabling issues initially?

Reply:
Changed to:

> This repo does not accept issues as of now. Please file issues to the appropriate development repos.
> For issues with the VMR itself, please use the [source-build repository](https://github.com/dotnet/source-build).